test/evergreen.yml

```diff
@@ -5658,7 +5666,7 @@ buildvariants:
     posix_configure_flags: -DENABLE_TCMALLOC=0
   tasks:
     - name: compile
-    - name: make-check-test
+    - name: make-check-test-debug
```

Comment:
Would it make more sense to set `CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O0` for the entire buildvariant?
If it helps with debugging `make-check-test` I assume it would also help with `unit-test-macos` and `fops` (`memory-model-test-mac` ignores these flags).

Reply:
Thanks, fixed in 68d2b35d48a5e27526b33c393416622c261d4517